cms/src/main/java/com/visitscotland/brxm/validator/EventDateValidator.java

```diff
@@ -38,11 +39,22 @@ public Optional<Violation> validate(ValidationContext context, Node node) {
         }
     }
 
-    private boolean isEndDateAfterStartDate(final Node node) throws RepositoryException {
+    private boolean isEventDatesValid(final Node node) throws RepositoryException {
         final Calendar startDate = getStartDate(node);
         final Calendar endDate = getEndDate(node);
 
-        return endDate.after(startDate);
+        return isCalendarDefaultDate(endDate) || endDate.after(startDate);
+    }
+
+    private static boolean isCalendarDefaultDate(Calendar calendar) {
```

Comment:
Let's keep this final to keep it consistent!

```suggestion
    private static boolean isCalendarDefaultDate(final Calendar calendar) {
```